ipa-core/src/net/server/handlers/query/status.rs

```diff
@@ -32,8 +32,11 @@ pub fn router(transport: Arc<HttpTransport>) -> Router {
 
 #[cfg(all(test, unit_test))]
 mod tests {
-    use axum::http::uri::{Authority, Scheme};
-    use hyper::{Body, StatusCode};
+    use axum::{
+        body::Body,
```

Comment:
Sorry for the delayed response, I wanted to investigate this a bit more. 

The thing is that the Axum Body is a really convenient type since it supports all our use cases for Empty, Full and Streaming types which are disjointed in Http Body. If we don't use it, we'll have to re-implement something very similar. Given that, I would rather not re-implement it now. If we ever want to move away from Axum, we can decide what we want to do at that point.

Reply:
So you're saying that it is not possible to use `Empty`, `Full` and `Streaming` directly in our code without extra code that needs to wrap them to fit our specific use case? I wouldn't re-implement axum functionality in this case, but perhaps my understanding that `axum::body` is just a wrapper around these was wrong